core-services/prow/02_config/openshift/aws-pod-identity-webhook/_prowconfig.yaml

```diff
@@ -51,6 +51,65 @@ tide:
     - needs-rebase
     repos:
     - openshift/aws-pod-identity-webhook
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - docs-approved
+    - lgtm
+    - px-approved
+    - qe-approved
```

Comment:
did you also add these new labels to your repo?

Reply:
nm, looks like we added them org wide
https://github.com/openshift/release/blob/a102cc64536e73bdab6463d2b54672f35b6e282d/core-services/prow/02_config/_labels.yaml#L429

Reply:
Dan checked before I opened this PR, and the labels where already in both repos.